flow360/component/simulation/cloud.py

```diff
@@ -0,0 +1,114 @@
+"""Cloud communication for simulation related tasks"""
+
+import json
+import time
+from datetime import datetime
+
+from flow360.cloud.rest_api import RestApi
+from flow360.component.case import Case
+from flow360.component.interfaces import DraftInterface, ProjectInterface
+from flow360.component.simulation.simulation_params import SimulationParams
+from flow360.component.simulation.web.asset_base import AssetBase
+from flow360.component.surface_mesh import SurfaceMesh
+from flow360.component.volume_mesh import VolumeMesh
+from flow360.exceptions import Flow360WebError
+from flow360.log import log
+
+TIMEOUT_MINUTES = 60
+
+
+def _check_project_path_status(project_id: str, item_id: str, item_type: str) -> None:
+    RestApi(ProjectInterface.endpoint, id=project_id).get(
+        method="path", params={"itemId": item_id, "itemType": item_type}
+    )
+    # pylint: disable=fixme
+    # TODO: check all status on the given path
+
+
+def _run(
+    starting_point: AssetBase,
+    params: SimulationParams,
+    destination: type[AssetBase],
+    async_mode: bool = True,
```

Comment:
can we call them:
```suggestion
    source_asset: AssetBase,
    params: SimulationParams,
    target_asset: type[AssetBase],
    async_mode: bool = True,
```
?

Reply:
Done